extensions/blocks/send-a-message/shared/countrycodes.js

```diff
@@ -0,0 +1,241 @@
+export const countryCodes = [
```

Comment:
This country codes' list troubles me a bit as it could become outdated. Maybe we could pull that list from somewhere else instead of having to update it in Jetpack? Maybe we could use a package here (maybe from Calypso or a third party plugin) or an API call to get the data from wpcom.

Reply:
New endpoint ready for review in D45372-code. :+1:

Reply:
On second thoughts, I don't think using this API endpoint is a good idea. It means blocking rendering of the UI until the endpoint resolves (bad on slow connections). It's also data that will very rarely change (the last change was years ago).